spec/fixtures/proofing/lexis_nexis/ddp/failed_response.json

```diff
@@ -0,0 +1,14 @@
+{
+  "account_address_result": "not found",
+  "account_email_result": "not found",
+  "account_name_result": "not found",
+  "account_telephone_result": "not found",
+  "request_id": "1234",
```

Comment:
for a JSON request body, I think this would be easier to check directly in the specs, instead of having to compare to a separate fixture file? (different for XML requests)